### ISSUE TITLE
Ratter

### ISSUE THREAD

**Issue by darkluxer** [raax7/OG-Fortnite-Cheat#26]
![IMG_1840](https://github.com/user-attachments/assets/482044e2-f369-4de4-90f3-7afd851d151d)
 I added this guys discord and he sent me his “”version of his cheat”” and he sent me a rat

**Comment by raax7** [comment]
SO BLOPOOCKY WOCKY HIM MATEY... SIMPLE EXPLANTION WOUDL BE BLOCK HIM

**Comment by letmecheatinfn** [comment]
yeah sent me a rat too. 36 out of 70 smth detections on virustotal is lowkey crazy
